Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM library/ubuntu:artful
+
+# Create app directory
+WORKDIR /usr/src/site
+
+# Install app dependencies
+COPY . .
```

Comment:
Put this *after* the prerequisite steps (`apt` operations/etc, before `npm`) so that changes don't invalidate the entire cache.